package.json

```diff
@@ -20,7 +20,6 @@
     "findup-sync": "^4.0.0",
     "fs-extra": "^8.1.0",
     "inquirer": "8.2.0",
-    "jest": "^29.5.0",
```

Comment:
Interesting that this has been hiding in the prod deps all this time.  It looks like we also have a couple `@types` deps in there as well.  Would you be ok moving those in this PR as well?

Reply:
done in [ad784bd](https://github.com/HubSpot/hubspot-cli/pull/1242/commits/ad784bd1e49be35f63a7c5a5eea61bc452d03d22)! I totally missed those